libs/sdk-ui-dashboard/src/model/commandHandlers/index.ts

```diff
@@ -123,7 +124,7 @@ export const DefaultCommandHandlers: {
     "GDC.DASH/CMD.INSIGHT_WIDGET.CHANGE_FILTER_SETTINGS": changeInsightWidgetFilterSettingsHandler,
     "GDC.DASH/CMD.INSIGHT_WIDGET.CHANGE_PROPERTIES": changeInsightWidgetVisPropertiesHandler,
     "GDC.DASH/CMD.INSIGHT_WIDGET.CHANGE_CONFIGURATION": changeInsightWidgetVisConfigurationHandler,
-    "GDC.DASH/CMD.INSIGHT_WIDGET.CHANGE_INSIGHT": notImplementedCommand,
+    "GDC.DASH/CMD.INSIGHT_WIDGET.CHANGE_INSIGHT": changeInsightWidgetInsightHandler,
```

Comment:
consider change name from changeInsightWidgetInsightHandler to changeInsightWidgetHandler